gsd/gsd.c

```diff
@@ -64,16 +64,16 @@ enum
     GSD_INITIAL_WRITE_BUFFER_SIZE = 1024
     };
 
-/// Maximum size of write buffer
+/// Default maximum size of write buffer
 enum
     {
-    GSD_MAXIMUM_WRITE_BUFFER_SIZE = 16 * 1024 * 1024
+    GSD_DEFAULT_MAXIMUM_WRITE_BUFFER_SIZE = 64 * 1024 * 1024
     };
 
-/// Size of copy buffer
+/// Default number of index entries to buffer
 enum
     {
-    GSD_COPY_BUFFER_SIZE = 128 * 1024
+    GSD_DEFAULT_INDEX_ENTRIES_TO_BUFFER = 262144
```

Comment:
Why the explicit number rather than `2 * 128 * 1024` like in the maximum buffer write.

Reply:
Only because I was experimenting with different power of 2 and typing them in directly. Will rewrite in a cleaner form.